lib/elixir/src/elixir_def_overridable.erl

```diff
@@ -32,6 +32,13 @@ name(_Module, {Name, _} = Function, Overridable) ->
   {Count, _, _, _} = maps:get(Function, Overridable),
   elixir_utils:atom_concat([Name, " (overridable ", Count, ")"]).
 
+%% Returns the kind (def, defp, and so on) of the given overridable function.
+
+kind(Module, FunctionOrMacro) ->
```

Comment:
We'd need to return the `FinalArgs` as well (cause we want to add the `'_@CALLER'` arg), this is why I didn't go with `kind_and_name` (which is what I wanted to go with for the record 😄). Wdyt?